dev_requirements.txt

```diff
@@ -8,3 +8,7 @@ flake8-docstrings
 
 flake8-tuple
 flake8-quotes
+
+django-extensions
```

Comment:
What are these new requirements for?

Reply:
This is a dev only extension to provide the "django-admin shell_plus" an Ipython based django shell for debugging.